message ConfigRequest
{
    enum Subsystem
    {
	MANAGER=1;
    }

    required Subsystem subsystem = 1;
}

